lambda/translprim.ml

```diff
@@ -902,8 +935,10 @@ let lambda_primitive_needs_event_after = function
   | Paddfloat _ | Psubfloat _ | Pmulfloat _ | Pdivfloat _ | Pstringrefs | Pbytesrefs
   | Pbox_float _ | Pbox_int _
   | Pbytessets | Pmakearray (Pgenarray, _, _) | Pduparray _
-  | Parrayrefu (_, (Pgenarray | Pfloatarray))
-  | Parraysetu (_, (Pgenarray | Pfloatarray))
+  | Parrayrefu (Pgenarray_ref _ | Pfloatarray_ref _)
+  | Parraysetu (Pgenarray_set _ | Pfloatarray_set)
+    (* CR aspectorzabusky for mshinwell: It seems to be that unsafe set should
+       never allocate or throw an exception; am I missing something? *)
```

Comment:
Agreed, it seems odd that these cases are here.

Reply:
Done on flambda-backend in 123629587